Sources/Datadog/Tracing/Integrations/TracingWithLoggingIntegration.swift

```diff
@@ -73,26 +80,23 @@ internal struct TracingWithLoggingIntegration {
             )
         }
 
-        let threadName = getCurrentThreadName()
-
-        core.v1.scope(for: LoggingFeature.self)?.eventWriteContext { context, writer in
-            let log = logBuilder.createLogEvent(
-                date: date,
-                level: level,
-                message: message,
-                error: extractedError,
-                attributes: .init(
-                    userAttributes: userAttributes,
-                    internalAttributes: internalAttributes
-                ),
-                tags: [],
-                context: context,
-                threadName: threadName
-            )
-
-            if let log = log {
-                writer.write(value: log)
-            }
-        }
+        core.send(
+            message: .custom(
+                key: "log",
+                attributes: [
+                    "date": date,
+                    "loggerName": configuration.loggerName,
+                    "service": configuration.service,
+                    "threadName": Thread.current.dd.name,
+                    "message": message,
+                    "level": level,
```

Comment:
We are still sharing types such as `LogLevel` here, this dependency will eventually be removed.